metricbeat/module/elasticsearch/stats/_meta/data.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "elasticsearch": {
+        "stats": {
+            "docs": {
+                "count": 4,
+                "deleted": 0
+            },
+            "segments": {
+                "count": 4,
+                "memory": {
+                    "bytes": 6647
+                }
+            },
+            "shards": {
+                "failed": 0,
+                "successful": 11,
+                "total": 22
+            },
+            "store": {
```

Comment:
Is this `store.size.bytes` the total disk space used by indices? If so, that's awesome, if not then I'm not sure we want it.